packages/autocomplete-plugin-tags/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@algolia/autocomplete-plugin-tags",
+  "description": "A plugin to manage and display a list of tags in Algolia Autocomplete.",
+  "version": "1.2.2",
```

Comment:
We'll need to update to the latest version.

Reply:
Ship.js will do it for us, won't it?

Reply:
See https://github.com/algolia/autocomplete/pull/645#discussion_r706090313.

Reply:
Addressed in 893d847.